esp-hal/doc-helper/before

```diff
@@ -1,12 +1,10 @@
 # #![no_std]
-# use esp_hal::peripherals::Peripherals;
-# use esp_hal::clock::ClockControl;
-# use esp_hal::system::SystemControl;
+# use esp_hal::prelude::*;
+# use procmacros::handler;
+# use esp_hal::interrupt;
 # #[panic_handler]
 # fn panic(_ : &core::panic::PanicInfo) -> ! {
 #     loop {}
 # }
+# #[allow(unused_variables)]
 # fn main() {
-#   let peripherals = Peripherals::take();
```

Comment:
I think the idea was to hide the init code and only show that in the top level docs
Having it in the snippet would have required less changes in the other places